guides/common/modules/proc_installing-puppet-agent-during-host-provisioning.adoc

```diff
@@ -49,11 +53,13 @@ ifndef::katello,orcharhino,satellite[]
 +
 Select the *boolean* type and set the value to `true`.
 endif::[]
-ifdef::katello,orcharhino,satellite[]
-. Enable the Puppet agent using a host parameter in global parameters, a host group, or for a single host.
-Add a host parameter named `enable-puppet7`, select the *boolean* type, and set the value to `true`.
-endif::[]
+. Set configuration for the Puppet agent.
+* If you use a non-integrated Puppet setup, set the following host parameters in global parameters or a host group:
```

Comment:
```suggestion
* If you use an external Puppet Server or setup, set the following host parameters in global parameters or a host group or while creating a host:
```

Reply:
I think as it is now, it's clear from the context.